src/github_runner_image_builder/builder.py

```diff
@@ -745,6 +758,31 @@ def _get_github_runner_version(version: str) -> str:
     return latest_version.lstrip("v")
 
 
+def _chown_home() -> None:
+    """Change the ownership of Ubuntu home directory.
+
+    Raises:
+        HomeDirectoryChangeOwnershipError: If there was an error changing the home directory
+        ownership to ubuntu:ubuntu.
+    """
+    try:
+        subprocess.check_call(
+            ["/usr/bin/chown", "--recursive", "ubuntu:ubuntu", "/home/ubuntu"],  # nosec
```

Comment:
Can we use a constant here for the user?

Reply:
Will do it on the next PR! Thanks :D